core/src/test/java/org/ehcache/EhcacheEventTest.java

```diff
@@ -82,12 +82,6 @@ public void setUp() throws Exception {
     cache.init();
   }
   
-  @After
-  public void tearDown() {
-    // Make sure no more events have been sent
-    verify(eventNotifier, new NoMoreInteractions()).onEvent(any(CacheEvent.class));
```

Comment:
Removing this check as this essentially checks for any interaction with `eventNotifier` which is not true anymore as `eventNotifier` might be invoked for `fireEvents()` method